framework/resources/Datasources/table_stats/analyze_tables.sql

```diff
@@ -0,0 +1,19 @@
+alter session set `store.parquet.reader.int96_as_timestamp`=true;
+analyze table `table_stats/alltypes_with_nulls` compute statistics;
+analyze table `table_stats/date_intervals` compute statistics;
+analyze table `table_stats/impala_parquet` compute statistics;
+--analyze table `table_stats/onlynulls_parquet` compute statistics;
```

Comment:
Any reason this is commented? May be remove it if it's not needed or add a brief comment above this line to explain why it's commented.

Reply:
There was no corresponding data src for this. I removed this line.